src/compat.c

```diff
@@ -0,0 +1,20 @@
+#include <postgres.h>
+#include <funcapi.h>
+
+/* Old functions that are no longer used but are needed for compatibiliy when
```

Comment:
It seems it is required to start previous version of DB with new shared lib, before functions are dropped.